tools/autograd/utils.py

```diff
@@ -92,12 +86,6 @@ def load_op_list_and_strip_overload(op_list, op_list_path):
     # strip out the overload part
     return {opname.split('.', 1)[0] for opname in op_list}
 
-def group_declarations_by_op_name(declarations):
```

Comment:
just moved this to gen_pyi.py and renamed to group_by_base_name since it no longer operates on Declarations entries